.github/workflows/build.yml

```diff
@@ -7,22 +7,38 @@ jobs:
         java-version: [1.8, 11, 14]
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
       - uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java-version }}
       - uses: eskatos/gradle-command-action@v1.3.2
         with:
           gradle-version: wrapper
-          arguments: check build publishToMavenLocal --stacktrace --parallel
+          arguments: remapAllJars compileTestmodJava --stacktrace --parallel
           dependencies-cache-enabled: false
           configuration-cache-enabled: false
       - uses: actions/upload-artifact@v2
         with:
           name: Artifacts
           path: build/libs/
       - uses: actions/upload-artifact@v2
+        # Technically this shouldn't cause issues, but the MC world runs on J8 and it's better safe than sorry.
+        if: ${{ matrix.java-version == 1.8 }}
```

Comment:
No, I would use j14/15 here, thats what I am planning on using to do the release builds.